test/cmd/command_test.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func TestRunCommand(t *testing.T) {
+	testCases := []struct {
+		command string
+		expectedOutput string
+		shouldGetError bool
+	}{
```

Comment:
Can we have a test for error output? That's an important piece of these helpers.

Reply:
I feel not like the tests for error output, since it's tied to the OS. And it will be confusing if the tests pass on Prow and failed when running on local (a different OS)

Reply:
Error output is not tied to the OS, but to the command you run. Use standard error codes to take care of that; for example, running the nonexisting command "foobar" returns the error code 127, or you can exit a specific error code. For example, if you run "bash -c 'echo foo > /dev/stderr; exit 4'" you will get the error code 4 and just "foo" in stderr no matter the OS, Prow or local.

Reply:
Got it. Added the tests.
Thanks.